parts/linux/cloud-init/artifacts/components.json

```diff
@@ -411,7 +411,7 @@
         "mariner": {
           "current": {
             "versions": [
-              "1.4.54"
+              "1.1.9-5.cm2"
```

Comment:
What does "cm2" stand for?

Reply:
I don't know either. But I got this version from a previous pipeline that we've been using.
`azure-arm:  moby-runc x86_64 1.1.9-5.cm2 mariner-official-base 3.4 M`